packages/design/src/getMobileShadows.js

```diff
@@ -36,7 +36,7 @@ function getShadowStyles(baseTokens) {
         height: baseTokens["space-minuscule"],
       },
       shadowOpacity: 0.16,
-      shadowRadius: baseTokens["radius-large"],
+      shadowRadius: baseTokens["radius-small"],
```

Comment:
@ZakaryH that makes sense. I just wanted to make sure that there wasn't anything that "made sense" to have a huge radius (probably not but worth noting)

Reply:
Late to the party but using border radius tokens to define a shadow value doesn't make any sense (I probably wrote this initially) - these should just be raw numbers and not linked in any way to how we define corner roundness!

Reply:
Also, did we test this on both Android and iOS? Shadows are expressed differently on each platform

Reply:
>Late to the party but using border radius tokens to define a shadow value doesn't make any sense (I probably wrote this initially) - these should just be raw numbers and not linked in any way to how we define corner roundness!

that's fair. I am not too sure of the rationale of doing it this way to begin with. maybe there's a compelling reason I'm not thinking of.

>Also, did we test this on both Android and iOS? Shadows are expressed differently on each platform

you betcha! we tried it out on Android and it makes no difference because `shadowRadius` is an iOS only prop

![image](https://github.com/GetJobber/atlantis/assets/11843215/7d6fd1bc-1b2b-4fb6-aa8f-16d4fab02096)

Reply:
beautiful 🙏 

I will _maybe_ throw up a cleanup PR to decouple these values from our border-radius tokens as an exercise in getting my mobile dev env running on my new machine!